lib/actions/deleteLinkFromProjectAction.js

```diff
@@ -0,0 +1,32 @@
+const assert = require('assert');
+const DeleteLinkFromProjectResponse = require('./deleteLinkFromProjectResponse');
+const ProjectLink = require('../projectLink');
+
+class DeleteLinkFromProjectAction {
+  constructor(projectLink = ProjectLink) {
+    this.projectLink = projectLink;
+  }
+
+  // eslint-disable-next-line class-methods-use-this
+  parseParams(userInputString = '') {
+    assert(typeof userInputString === 'string');
```

Comment:
Asserts are usually used just for testing. I believe we carried this over from a previous action. Do you know what it's purpose is here?

Reply:
This is a way that we can test to assert that the type of input from the user is a string. 
It's easy to test and just another way. We don't have to use this.